app/repositories/CacheRepository.scala

```diff
@@ -45,7 +45,7 @@ class CacheRepositoryImpl @Inject()(
 
   override def idIndex: Aux[BSONSerializationPack.type] = Index.apply(BSONSerializationPack)(
     key = Seq("internalId" -> IndexType.Ascending),
-    name = Some("internal-id-index"),
+    name = Some("internal-auth-id-index"),
```

Comment:
We want to log the existing indexes (as they are in Mongo) just once, and if we log it at warn level, then we won't have to do anything additional in production to make them appear.

In this class, I would add something like the following:

```
  import play.api.Logger
  private val logger = Logger(this.getClass)

  // Temporarily adding to log index names once
  private lazy val logIndexesOnce: Unit =
    collection.flatMap(_.indexesManager.list()).foreach { indexes =>
      val indexNames = indexes.flatMap(_.name)
      logger.warn(s"[CacheRepository]: Index Names ${indexNames.mkString(",")}")
      logger.warn(s"[CacheRepository]: Indexes ${indexes.mkString(",")}")
    }

  override def get(internalId: String): Future[Option[MatchingAndSuitabilityUserAnswers]] = {
    logIndexesOnce 
    findCollectionAndUpdate[MatchingAndSuitabilityUserAnswers](selector(internalId))
  }
  ```

Reply:
note that `get` is an existing method, we're just adding the `logIndexesOnce` line to make the log appear